proto-wine/ftnoir_protocol_wine.cpp

```diff
@@ -74,9 +75,13 @@ module_status wine::initialize()
         QString proton_path(const QString& proton_path);
 
         wine_path = proton_path(s.proton_path().toString());
-        env = make_steam_environ(s.proton_path().toString(), s.proton_appid);
+        try {
```

Comment:
Exception handling is disallowed on project level.

Reply:
Just use something like

```c++
using t = std::tuple<QString, QString, bool>;
auto [path, error_string, success] = ...;
if (!success)
    ...;
```

Reply:
Thank you, that was exactly enough information for me to efficiently re-write this.